app/channel.go

```diff
@@ -478,7 +478,8 @@ func (a *App) CreateGroupChannel(userIds []string, creatorId string) (*model.Cha
 	return channel, nil
 }
 
-func (a *App) createGroupChannel(userIds []string, creatorId string) (*model.Channel, *model.AppError) {
+// `creatorId` is replaced with _ to pass unused param lint
+func (a *App) createGroupChannel(userIds []string, _ string) (*model.Channel, *model.AppError) {
```

Comment:
These are all private methods. I don't believe it needs to satisfy any interface. Why can't we just remove the parameter?

Reply:
I'll do it next. It's my first task in WIP list.

Reply:
Woops, my bad there. Keep going!